### ISSUE TITLE
TimerTask uses deprecated API

### ISSUE THREAD

**Issue by sschepens** [ruby-concurrency/concurrent-ruby#341]
I'm using TimerTask and it's generating lot's of these logs:
```
2015-06-11T14:12:29Z Pid:9764 Actor:Module Level:WARN: [DEPRECATED] `Concurrent.timer` is deprecated and it'll removed in next release, use `ScheduledTask` instead
called on: concurrent-ruby-0.9.0.pre2/lib/concurrent/timer_task.rb:300:in `schedule_next_task'
```

**Comment by jdantonio** [comment]
I'll take a look and try to reduce the noise.

**Comment by jdantonio** [comment]
@sschepens Yep. That's a bug. When I deprecated `Concurrent.timer` I failed to notice that I was calling that method in `TimerTask`. I'll work on this ASAP and it will be fixed in the full 0.9 release.

**Comment by jdantonio** [comment]
Fixed in #344. Will be released in v0.9.0.
